recipes/freeimage/all/patches/003_link_conan.patch

```diff
@@ -0,0 +1,155 @@
+--- Source/FreeImage/J2KHelper.cpp
```

Comment:
If it's not too much trouble, can you upstream this patch?
I think the biggest trouble will be modifying all makefiles to add the include paths.